shells/web-shell/README.md

```diff
@@ -1,28 +1,38 @@
-## web-shell Flags
+# web-shell Flags
 
-e.g. https://live.arcs.dev/shells/web-shell/?storage=pouchdb&user=barney
+e.g. https://live.arcs.dev/shells/web-shell/?log&storage=$firebase/scott
 
-### Setting Storage Provider
+## Setting User-Persona/Storage-Key
+
+### Today, the User-Persona is identified exactly by a Storage-Key: you are where your stuff is.
+
+* persona=[storage-key]
+  * use `storage-key`
+
+#### To create a custom User-Persona for one of the existing servers, macros are provided:
+
+* persona=$firebase/[name]
+  * appends [name] to the arcs-owned firebase instance key, e.g. firebase://arcs-storage.firebaseio.com.../[name]
+* persona=$pouchdb/[name]
+  * appends [name] to the default pouchdb key, e.g. pouchdb://local/arcs/[name]
+
+#### other options:
 
 * no flag
   * use the same storage as last time, otherwise `default`
-* storage=pouchdb
+* persona=pouchdb
```

Comment:
what does a persona with no `[name]` mean?

Reply:
I'm still having trouble explaining this thing, maybe you can help me communicate it better. The `name` really means nothing, I only talk about `name` so people can figure out how to replicate what they did before. Bottom line: you need a database location that is unique to you. For our shared Firebase instances, appending a name or an ldap is an easy way to establish such a location. If you have your own personal Firebase instance: that key doesn't need to have a name in it anywhere.